scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +153,37 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeat amount of times.
+    """
+
+    def __init__(self,
+                 operation: str,
+                 configuration: Dict,
+                 repetitions: int = 0) -> None:
```

Comment:
`0` should be "no iteration", `-1` should be infinite.

Reply:
The iteration should default to "infinite" iteration, not to no iteration. At least according to the design doc. If you want to change it feel free to leave a comment or a suggestion.